pkg/controller/directimagestreammigration/copyimage.go

```diff
@@ -0,0 +1,104 @@
+package directimagestreammigration
+
+import (
+	"errors"
+
+	"github.com/containers/image/v5/copy"
+	"github.com/containers/image/v5/types"
+	"github.com/konveyor/mig-controller/pkg/compat"
+	"github.com/konveyor/openshift-velero-plugin/velero-plugins/imagecopy"
+)
+
+func (t *Task) migrateInternalImages() error {
+	imageStream, err := t.Owner.GetImageStream(t.Client)
+	if err != nil {
+		return err
+	}
+	if imageStream == nil {
+		return errors.New("ImageStream not found")
```

Comment:
I would actually return and apiserver NotFound error here, with object imagestream name and namespace embeded

Reply:
well after thinking some more, this does not make much sense, we are catching the NotFound error in GetImageStream function and then creating a new error here. If the aim is to bubble up the NotFound error, I wonder if we should return the original error itself?

Reply:
Good point. I was following the prior pattern, which makes sense for a general-purpose "find" method where not finding it isn't an error and `nil` is what you want, but I'm only using this in one place, and here, "not found" is an error, so I should probably skip the checking for IsNotFound there, and then here I won't need to check for nil at all.

Reply:
Ive updated `GetImageStream` to propagate all errors, including not found errors, so I just removed this nil check and error message return.